src/main/java/pl/tymoteuszboba/emojimize/parser/EmojiParser.java

```diff
@@ -0,0 +1,33 @@
+package pl.tymoteuszboba.emojimize.parser;
+
+import pl.tymoteuszboba.emojimize.api.VendorApi;
+import pl.tymoteuszboba.emojimize.entity.emoji.Emoji;
+
+import java.io.IOException;
+import java.util.Optional;
+
+public class EmojiParser {
+
+    public static String parse(VendorApi api, String originalString) {
+        String[] splittedPhrase = originalString.split(" ");
+
+        for (String word : splittedPhrase) {
+            if (word.startsWith(":") && word.endsWith(":")) {
+                String wordCopy = word;
+                word = word.replace(":", "");
```

Comment:
Having a single instance of the `word` variable here would seem like a better idea, and so I would rename it into something more specific like '`magicWord`' and instead of making yet another copy, introduce a `word` variable which would then have the colons replaced, leaving the former `magicWord` intact and used further down in a state it's been in since its initialisation

Reply:
Fine thing! I'll change `wordCopy` into a better name and refactor it quite a bit.

Reply:
The suggestion has been resolved in the latest commit.